src/Microsoft.SqlTools.ServiceLayer/Scripting/ScripterCore.cs

```diff
@@ -286,7 +286,7 @@ internal DefinitionResult GetDefinitionUsingDeclarationType(DeclarationType type
             string createSyntax = null;
             if (objectScriptMap.ContainsKey(objectType.ToLower()))
             {
-                createSyntax = string.Format("CREATE {0}", objectScriptMap[objectType.ToLower()]);
+                createSyntax = string.Format("CREATE");
```

Comment:
This will create a problem because the scripts contain comments on top which have the word "create". I believe @kevcunnane talked about this bug last time, and that was the reason I added the objectType in the search string after "create".

Reply:
yes but keeping the code causes returning empty string. sometimes there are more than one space between create and type so at least this way the script is returned but the cursor is in the wrong place

Reply:
Perhaps we can have a check to have the word "create" as well as the objectType in the same line, rather than look for the string ("Create {0}", objectType)